kuvaselaamo/settings.py

```diff
@@ -4,6 +4,17 @@
 import os
 import environ
 
+checkout_dir = environ.Path(__file__) - 2
+assert os.path.exists(checkout_dir("manage.py"))
+
+parent_dir = checkout_dir.path("..")
+if os.path.isdir(parent_dir("etc")):
+    env_file = parent_dir("etc/env")
+    default_var_root = environ.Path(parent_dir("var"))
```

Comment:
Is this actually used somewhere?

Reply:
Those use it to set a default value for `STATIC_ROOT` https://github.com/City-of-Helsinki/palvelutarjotin/blob/master/palvelutarjotin/settings.py#L24. This project does not do that, at least for now 🙂